### PR TITLE
added Triage document for weekly triages

### PR DESCRIPTION
**Pull request by cgchinmay**
### Description
[Describe what this change achieves]
This PR is to start gathering feedback and update triage.md

### Testing
NA

[Describe how this change was tested]
rendered the markdown in local setup

---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: triage.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- [Bugs](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
+    - Search for Issues which are labeled as bugs
+    - Provide status update such as In Progress, Not Planned, Need more information.
+
+- [Enhancements](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-desc+label%3Aenhancement+)
+    - Search for feature requests/enhancements. Provide status update and assign owners if its going to be next release candidate
+
+
+*There could be overlap of an Issue in more than 1 category such as Breaking and Bugs or Breaking and High Priority. In this case triage only once and pick another issue. Also ensure that*
```
Comment (IanHoang) [inline triage.md:40]: Is there a sentence missing at the end? "Also ensure that"

File: triage.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Every week a maintainer will be assigned on a rotational basis to conduct triage meeting.
+The meeting duration will be no more than 1 hour every Tuesday @1.30 PM PDT/3.30 PM CDT
+
+Meeting link: []
+
+## What is Triaging
+Triage is a process of grooming issues with correct labels, assigning them owners if available, ensuring that the issues have all the required information so that they are actionable
+
+## How to Triage
+The below steps are not exhaustive and can be extended. These will be performed for opensearch-benchmark as well as opensearch-benchmark-workloads repositories. In every triage, we should try to cover atleast 10 Issues combined from below categories
```
Comment (IanHoang) [inline triage.md:20]: Should we have dual links for both repositories? For example for each type of issue, we have links to both repos?

**Review by IanHoang — CHANGES_REQUESTED** [review]
Left some comments

File: triage.md (hunk 1 above)
Comment (cgchinmay) [reply triage.md:40]: Let me update

File: triage.md (hunk 2 above)
Comment (cgchinmay) [reply triage.md:20]: Could you elaborate. I did not understand

Comment (gkamat) [reply triage.md:20]: atleast -> at least

Also, embed links to the two repositories in this sentence.

File: triage.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+- [Untriaged](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aopen+is%3Aissue+no%3Alabel) Issues
+    - Search for Issues which do not have any labels sorted by newest to oldest
+    - Check if there are any duplicates, if yes then comment the duplicate issue link and close one of the two
```
Comment (gkamat) [inline triage.md:24]: Indicate what is to be done with untriaged issues -- analyze, categorize, assign, or tag the appropriate SME, etc.

File: triage.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+- [High Priority](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aopen+is%3Aissue+label%3A%22High+Priority%22)
+    - Search for High Priority Issues. Ensure they have a status update or an owner
+
+- [Breaking](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aopen+is%3Aissue+label%3Abreaking)
+    - Search for Issues which are labeled as breaking. Try to assign owners if its high priority beaking change
```
Comment (gkamat) [inline triage.md:30]: if it is a high..

File: triage.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+- [Bugs](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
+    - Search for Issues which are labeled as bugs
+    - Provide status update such as In Progress, Not Planned, Need more information.
+
+- [Enhancements](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-desc+label%3Aenhancement+)
+    - Search for feature requests/enhancements. Provide status update and assign owners if its going to be next release candidate
```
Comment (gkamat) [inline triage.md:37]: if it is going to be the next release ...

**Review by gkamat — CHANGES_REQUESTED** [review]

File: triage.md (hunk 2 above)
Comment (IanHoang) [reply triage.md:20]: Right now, the headers -- such as Untriaged, High Priority, Breaking, etc. --  are currently only pointing to OSB repository filtered issues. I was suggesting that we should include links pointing to filtered issues for OSB workloads repository as well

Comment (cgchinmay) [reply triage.md:20]: Got it, will update for OSB workloads as well

File: triage.md (hunk 3 above)
Comment (cgchinmay) [reply triage.md:24]: Thanks for the feedback. I will make the necessary changes

File: triage.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+## Scope
+These guidelines serve as a primary document for triaging issues in [opensearch-benchmark](https://github.com/opensearch-project/opensearch-benchmark) and [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) project. For maintainers/contributors, it is a guide to ensure that we address all customer issues and address feature requests in a timely manner.
+
+## Schedule
+Every week a maintainer will be assigned on a rotational basis to conduct triage meeting.
+The meeting duration will be no more than 1 hour every Tuesday @1.30 PM PDT/3.30 PM CDT
+
+## What is Triaging
+Triage is a process of grooming issues with correct labels, assigning them owners if available, ensuring that the issues have all the required information so that they are actionable
```
Comment (gkamat) [inline triage.md:15]: Triaging is the process of prioritizing issues to be acted on, grooming them with correct labels...

File: triage.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
+## Table of Contents
+- [Scope](#scope)
+- [Schedule](#schedule)
+- [What Is Triaging](#what-is-triaging)
+- [How to Triage](#how-to-triage)
+
+## Scope
+These guidelines serve as a primary document for triaging issues in [opensearch-benchmark](https://github.com/opensearch-project/opensearch-benchmark) and [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) project. For maintainers/contributors, it is a guide to ensure that we address all customer issues and address feature requests in a timely manner.
```
Comment (gkamat) [inline triage.md:8]: This document provides guidelines for the triaging of GitHub issues in the ... and ... repositories.

File: triage.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+## Schedule
+Every week a maintainer will be assigned on a rotational basis to conduct triage meeting.
+The meeting duration will be no more than 1 hour every Tuesday @1.30 PM PDT/3.30 PM CDT
+
+## What is Triaging
+Triage is a process of grooming issues with correct labels, assigning them owners if available, ensuring that the issues have all the required information so that they are actionable
+
+## How to Triage
+The steps listed below are not exhaustive and can be updated. Start with [opensearch-benchmark](https://github.com/opensearch-project/opensearch-benchmark) followed by [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) project. For each of these projects we can follow the below steps to triage Issues from different categories. Try to cover at least 1 Issue from each of the below listed category.
```
Comment (gkamat) [inline triage.md:18]: project/projects -> repository/repositories

For each of these projects we can follow the steps below to triage issues from different categories. Try to cover at least one issue from each of the below listed categories.

File: triage.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+### Steps to triage [opensearch-benchmark](https://github.com/opensearch-project/opensearch-benchmark)
+- [Untriaged](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aopen+is%3Aissue+no%3Alabel)
+    - First check if any of the untriaged issue is a duplicate. If yes, then comment the duplicate Issue link and close the duplicate
+    - Assign appropriate labels for these Issues such as bug, enhancements, breaking etc.
```
Comment (gkamat) [inline triage.md:23]: Assign a priority label to the issue, if appropriate.

File: triage.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+    - If possible assign owners who could be [SME](https://en.wikipedia.org/wiki/Subject-matter_expert) for the Issue or volunteers if any
+
+- [Breaking](https://github.com/opensearch-project/opensearch-benchmark-workloads/issues?q=is%3Aopen+is%3Aissue+label%3Abreaking)
+    - Search for Issues which are labeled as breaking
+    - Try to assign owners if it is a high priority beaking change
+
+- [Bugs](https://github.com/opensearch-project/opensearch-benchmark-workloads/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
+    - Search for Issues which are labeled as bugs
+    - Comment with a status update such as In Progress, Not Planned, Needs more information
```
Comment (gkamat) [inline triage.md:58]: Add a priority label if the issue warrants it.

File: triage.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+- [Enhancement](https://github.com/opensearch-project/opensearch-benchmark-workloads/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement)
+    - Search for Issues which are feature requests/enhancements
+    - Comment with a status update and assign owners if it is going to be the next release candidate
```
Comment (gkamat) [inline triage.md:62]: Add comments as needed and ensure an owner is assigned if the feature is scheduled for the next release.

File: triage.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+- [Documentation](https://github.com/opensearch-project/opensearch-benchmark-workloads/issues?q=is%3Aopen+is%3Aissue+label%3Adocumentation)
+    - These Issues although not breaking are important for better customer experience
+    - Feel free to encourage the requester to raise a PR, these can be tagged as good first Issue as well
+
+*There could be an overlap of Issues in more than 1 category such as Breaking and Bugs or Breaking and High Priority. In this case try to triage a different Issue which is not already covered*
```
Comment (gkamat) [inline triage.md:68]: Notwithstanding the steps laid out above, some issues may fall into multiple categories and need to be triaged only once.

**Review by gkamat — CHANGES_REQUESTED** [review]

**Review by gkamat — APPROVED** [review]

**Pull request merged**